sdk/keyvault/keyvault-keys/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History
+
+## 4.0.0b1 (2019-06-28)
```

Comment:
Is this the correct version number for the keyvault libraries?

Reply:
Most probably isn't. Let me ask over teams.

Reply:
I just checked in package.json, I'd use `4.0.0-preview.1` explicitly here

Reply:
Done!